src/core/MemoryStream.h

```diff
@@ -58,7 +58,7 @@ class MemoryStream final : public IStream
     uint64  GetLength()                               const override;
     uint64  GetPosition()                             const override;
     void    SetPosition(uint64 position)                    override;
-    void    Seek(sint64 offset, int origin)                 override;
+    void    Seek(sint64 offset, sint32 origin)                 override;
```

Comment:
Doesn't the clang-format tool fix this automatically? There are many more of those cases.